### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/webkit2gtk-rs/issues/48

WIP as need include and use last gir, also contain [patch] section in Cargo.toml

### CODE REVIEW

File: src/auto/frame.rs [hunk 1]
```diff
@@ -22,11 +22,11 @@ pub const NONE_FRAME: Option<&Frame> = None;
 pub trait FrameExt: 'static {
     //#[cfg_attr(feature = "v2_22", deprecated)]
     //#[cfg(any(feature = "v2_2", feature = "dox"))]
-    //fn get_javascript_context_for_script_world<P: IsA<ScriptWorld>>(&self, world: &P) -> /*Ignored*/Option<java_script_core::Context>;
+    //fn get_javascript_context_for_script_world<P: IsA<ScriptWorld>>(&self, world: &P) -> /*Ignored*/Option<java_script_core::GlobalContextRef>;
```
Comment (antoyo) [inline src/auto/frame.rs]: Isn't this supposed to be generated with the new javascriptcore?

**Comment by antoyo** [comment]
One question, but the rest looks good to me.

File: src/auto/frame.rs (hunk 1 above)
Comment (EPashkin) [reply src/auto/frame.rs]: webkit2gtk-webextension-rs don't have dependence for javascriptcode, only -sys has.
Using these classes need many changes in javascriptcore and testing,
currently only stubs added.

**Comment by antoyo** [comment]
Can I merge this one as well?

**Comment by EPashkin** [comment]
No, it still contains path and waiting for @vhdirk's PR fixed and merged

**Comment by antoyo** [comment]
@EPashkin: Do I have anything else to review/merge or should I wait for @vhdirk?

**Comment by EPashkin** [comment]
IMHO not

**Comment by vhdirk** [comment]
Anything still needed from me?

**Comment by EPashkin** [comment]
@vhdirk Seems you fixed your PR.
I missed that sorry.
Then only my work needed.

**Comment by EPashkin** [comment]
I will apply https://github.com/gtk-rs/webkit2gtk-webextension-rs/pull/41#discussion_r268943900 yourself

**Comment by EPashkin** [comment]
Was no mail about force-pushing in #41 day ago.

**Comment by vhdirk** [comment]
Yes, I've been rather busy, so I didn't get to leaving a comment. Sorry about that.

**Comment by EPashkin** [comment]
@antoyo Updated PR but it still WIP, at minimum need build test for `example`,
currently not sure what better use workspaces or `.cargo\config [build]target-dir` for reuse already build crates

**Comment by EPashkin** [comment]
@antoyo, @vhdirk Added workspace to build `example` too.
As CI passed PR is ready to merge.

**Pull request merged**